R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1173,7 +1173,7 @@ test_that("group by, agg functions", {
 
   expect_equal(3, count(mean(gd)))
   expect_equal(3, count(max(gd)))
-  expect_equal(30, collect(max(gd))[1, 2])
+  expect_equal(30, collect(max(gd))[2, 3])
```

Comment:
I'm not sure if I understand this test correctly, looks like it's trying to test the second row of the result of `max(gd)` to see if it's 30.